README.md

```diff
@@ -1,80 +0,0 @@
-<p align="center">
```

Comment:
Same as changelog.

Reply:
It's symlinked, it will show normally. Check it out in my branch: https://github.com/iTwin/iTwinUI/tree/mayank/monorepo

Reply:
Glad that github understands unix symlinks because windows doesn't 😀
![image](https://user-images.githubusercontent.com/36186912/170502291-daf000ac-1fac-43db-a1fe-89078d02e3da.png)

Reply:
It's a symlink that github recognizes. What's the problem?

Reply:
That it does not work on my machine (and most of users).

Reply:
![image](https://user-images.githubusercontent.com/81580355/170982901-bac8487b-215f-4ff0-8057-61125f247a36.png)

It shows those linked folders images and if you press on files, you get nothing. This is expected too?
![image](https://user-images.githubusercontent.com/81580355/170983032-379df45c-6c63-443f-8414-3750d3413f3a.png)

Reply:
Hmm, so it won't matter in the case of `README` and `LICENSE` because github knows how to display those, but this `CHANGELOG` is not helpful. Should I just move it from root into the package? I'd imagine itwinui-variables will have its own `CHANGELOG`

Reply:
I dont know.. I see mui has a single changelog file in root with all packages listed in there. Not sure I am fan of that..
If we move changelog per package, how our `Releases` tab would look like? I like having it there.
I would say `LICENSE` and `README` keep in root. License copy from there to package on build and for separate pcakge we need to have separate `README`, while the root one could have general info ?

Reply:
Ok I'm duplicating the files now